pwiz_tools/Skyline/Controls/Graphs/PaneProgressBar.cs

```diff
@@ -31,46 +31,41 @@ public class PaneProgressBar : IProgressBar
         readonly LineObj _left = new LineObj()
         {
             IsClippedToChartRect = true,
-            Line = new Line() { Width = 4, Color = Color.Green, Style = DashStyle.Solid },
-            Location = new Location(0, 0, CoordType.PaneFraction),
+            Line = new Line { Width = 4, Color = Color.Green, Style = DashStyle.Solid },
+            Location = new Location(0, 0, CoordType.XPaneFractionYChartFraction),

```

Comment:
It would be better to be able to place this precisely aligned to the top of the graph the way we do for the RT stick annotations (ID and Predicted) in the chromatogram graphs.